src/FsAutoComplete/LspServers/AdaptiveFSharpLspServer.fs

```diff
@@ -359,7 +362,7 @@ type AdaptiveFSharpLspServer
         lspClient.NotifyDocumentAnalyzed
           { TextDocument =
               { Uri = filePath |> Path.LocalPathToUri
-                Version = version } }
+                Version = defaultArg version 0 } }
```

Comment:
adaptive server doesn't have a version all the time - why is this?

Reply:
I think I just modelled LSP Text document on being optional too. May also be when we need to read files from disk, but we can set that version to 0.

Reply:
Ok, I'll look at doing this

Reply:
just pushed up this change - essentially all input into the file system set a 0 version now.